.github/workflows/karma.yml

```diff
@@ -174,6 +174,8 @@ jobs:
                   accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
                   tunnelName: ${{ env.SAUCE_TUNNEL_ID }}
 
+            - run: DISABLE_STATIC_CONTENT_OPTIMIZATION=1 yarn sauce:ci
+            - run: DISABLE_STATIC_CONTENT_OPTIMIZATION=1 DISABLE_SYNTHETIC=1 yarn sauce:ci
```

Comment:
Probably needs a re-balancing at some point.

Reply:
🤔 Can jobs access/modify a shared global state? If we just had a list of flag combos and a pool of runners, then they'd auto balance...

Reply:
🤷